routes/route_integrations.jsx

```diff
@@ -18,7 +18,7 @@ export default {
             path: 'incoming_webhooks',
             indexRoute: {
                 getComponents: (location, callback) => {
-                    System.import('components/integrations/components/installed_incoming_webhooks').then(RouteUtils.importComponentSuccess(callback));
+                    System.import('components/integrations/components/installed_incoming_webhooks/index.js').then(RouteUtils.importComponentSuccess(callback));
```

Comment:
I get an error if I remove `/index.js` which is how I had in my PR before on mattermost-server repo. 

`ERROR in ./components/integrations/components/installed_incoming_webhooks.jsx
Module build failed: Error: ENOENT: no such file or directory, open '/Users/sudheer/go/src/github.com/mattermost/mattermost-webapp/components/integrations/components/installed_incoming_webhooks.jsx'`

Reply:
After updating master and cleaning my repo it started to work fine without the file extension. So I reverted this change.